src/setup_payload/AdditionalDataPayloadGenerator.cpp

```diff
@@ -45,8 +45,8 @@ using namespace chip::Encoding::LittleEndian;
 using chip::Encoding::BytesToUppercaseHexString;
 
 CHIP_ERROR
-AdditionalDataPayloadGenerator::generateAdditionalDataPayload(uint16_t lifetimeCounter, const char * serialNumberBuffer,
-                                                              size_t serialNumberBufferSize, PacketBufferHandle & bufferHandle,
+AdditionalDataPayloadGenerator::generateAdditionalDataPayload(uint16_t lifetimeCounter, const ByteSpan & uniqueId,
```

Comment:
s/uniqueId/rotatingDeviceUniqueId/ to make it clear this is relating to rotating device id.